org.emilien.SpaceLaunch

```diff
@@ -0,0 +1,42 @@
+{
+    "app-id" : "org.emilien.SpaceLaunch",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "spacelaunch",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "spacelaunch",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.com/elescoute/spacelaunch",
```

Comment:
when I try to view that project on gitlab, after being asked to login, it's 404. So either it's not the right URL or it is private and in either case, the build will fail. Also usually the checkout url end in `.git`